cobalt/black_box_tests/black_box_cobalt_runner.py

```diff
@@ -48,16 +48,26 @@ def __init__(self,
                url,
                log_file=None,
                target_params=None,
-               success_message=None):
+               success_message=None,
+               poll_until_wait_seconds=POLL_UNTIL_WAIT_SECONDS,
```

Comment:
👍 parametrizing timeouts is always good tweak, helps testability. (No action needed)